site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -208,6 +208,51 @@ The number of days before a child account that has not yet been verified by a pa
 +
 Required when the [field]#Delete unverified children# toggle is enabled.
 
+==== Multi-Factor
+
```

Comment:
Is it too much to add the premium edition blurb here?

```
include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
```

On the API docs update I did that and mentioned that authenticator based MFA was in the community edition.

Reply:
We also do this for the admin registration forms, so should be consistent.

Reply:
Are you suggesting using the `include` and then additionally clarifying that authenticator based MFA *IS* in the community edition and email/sms are not?  Could you point me to the example where you did that?  Or the branch if its checked in?

Reply:
https://github.com/FusionAuth/fusionauth-site/pull/578/files#diff-d24288277ae84845c927fc50297a3d26947802027907bce39958e0c51806963e

It's line 16 of `two-factor.adoc`.

We could optially pull it into a separate include file if we want (`totp-premium-blurb.adoc` or something).

Reply:
I'm not seeing anything in that pull request - is that the correct link?

Regardless, I added the default blurb to the authenticator field.

Reply:
I think you might need to expand `site/docs/v1/tech/apis/two-factor.adoc`, the diff is large enough it is hidden by default.

Reply:
🤦 ...thanks.
My head is still in a fog!